### PR TITLE
docs: update terraform

### PR DESCRIPTION
**Pull request by olegsu**
**What does this PR do?**
* Update terraform doc with more examples

### CODE REVIEW

**Review by eyalkraft — APPROVED** [review]

File: deploy/cloud/README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Delete environment
 
 ## Spesific version
```
Comment (gurevichdmitry) [inline deploy/cloud/README.md:28]:
```suggestion
## Specific version
```

File: deploy/cloud/README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ Delete environment
...
 To create an environment with spesific version use `terraform apply --auto-approve -var="stack_version=8.5.1"`
```
Comment (gurevichdmitry) [inline deploy/cloud/README.md:29]:
```suggestion
To create an environment with specific version use 

`terraform apply --auto-approve -var="stack_version=8.5.1"`
```

File: deploy/cloud/README.md [hunk 3]
```diff
@@ -27,6 +27,8 @@ Delete environment
...
+When working with non production versions it is required to also update the deployment regions.
+For example, to deploy `8.6.0-SNAPSHOT` use `terraform apply --auto-approve -var="stack_version=8.6.0-SNAPSHOT" -var="ess_region=gcp-us-west2"`
```
Comment (gurevichdmitry) [inline deploy/cloud/README.md:31]:
```suggestion
For example, to deploy `8.6.0-SNAPSHOT` use 

`terraform apply --auto-approve -var="stack_version=8.6.0-SNAPSHOT" -var="ess_region=gcp-us-west2"`
```

File: deploy/cloud/README.md [hunk 4]
```diff
@@ -27,6 +27,8 @@ Delete environment
...
 
 ## Named environment
 To give your environment a different prefix in the name use `terraform apply --auto-approve -var="deployment_name_prefix=elastic-deployment"`
```
Comment (gurevichdmitry) [inline deploy/cloud/README.md:34]:
```suggestion
To give your environment a different prefix in the name use 

`terraform apply --auto-approve -var="deployment_name_prefix=elastic-deployment"`
```

**Review by gurevichdmitry — CHANGES_REQUESTED** [review]
@olegsu, LGTM. Please fix some typo issues.

**Review by gurevichdmitry — APPROVED** [review]
LGTM

**Pull request merged**